core/java/iesi-core/src/main/java/io/metadew/iesi/metadata/definition/component/trace/componentDesign/HttpComponentQueryDesign.java

```diff
@@ -0,0 +1,21 @@
+package io.metadew.iesi.metadata.definition.component.trace.componentDesign;
+
+import lombok.Builder;
+import lombok.Data;
+
+import java.util.UUID;
+
+@Data
+public class HttpComponentQueryDesign  {
+    @Builder
+    public HttpComponentQueryDesign(UUID id,HttpComponentQueryDesignKey httpComponentQueryDesignID, String name, String value) {
```

Comment:
HttpComponentDesignTraceKey should be ComponentDesignTraceKey,
id should be a HttpComponentQueryParameterDesignTraceKey